ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/ForkChoiceUtil.java

```diff
@@ -49,9 +49,9 @@ public static Store get_genesis_store(BeaconStateWithCache genesis_state) {
     Bytes32 root = genesis_block.signing_root("signature");
     Checkpoint justified_checkpoint = new Checkpoint(UnsignedLong.valueOf(GENESIS_EPOCH), root);
     Checkpoint finalized_checkpoint = new Checkpoint(UnsignedLong.valueOf(GENESIS_EPOCH), root);
-    HashMap<Bytes32, BeaconBlock> blocks = new HashMap<>();
-    HashMap<Bytes32, BeaconState> block_states = new HashMap<>();
-    HashMap<Checkpoint, BeaconState> checkpoint_states = new HashMap<>();
+    ConcurrentHashMap<Bytes32, BeaconBlock> blocks = new ConcurrentHashMap<>();
```

Comment:
Good catch.

Reply:
Thanks, but this change came from your original PR. :)